core/tests/snapshot-recovery-test/tests/snapshot-recovery.test.ts

```diff
@@ -34,6 +35,12 @@ interface StorageLog {
     readonly enumerationIndex: number;
 }
 
+interface TokenInfo {
+    // FIXME: why isn't case changed?
```

Comment:
Unlike other DTOs return by Web3 RPCs, `TokenInfo` doesn't use `#[serde(rename_all = "camelCase")]`. Is it intentional? Since `TokenInfo` is also returned by `get_confirmed_tokens`, changing casing wouldn't be quite easy (unless `get_confirmed_tokens` is removed 🙃).